x-pack/plugins/apm/public/components/shared/charts/TransactionCharts/index.tsx

```diff
@@ -59,14 +59,18 @@ const ShiftedEuiText = styled(EuiText)`
   top: 5px;
 `;
 
-export function getResponseTimeTickFormatter(formatter: TimeFormatter) {
-  return (t: number) => formatter(t).formatted;
+export function getResponseTimeTickFormatter(formatter?: TimeFormatter) {
+  return (t: number) => {
+    return formatter ? formatter(t).formatted : asDuration(t);
+  };
```

Comment:
I feel like it's becoming quite unclear what formatters we are using where. Is there any way we can use the same formatter everywhere?

Reply:
I can remove the `formatter` parameter, I only kept it because `public/components/shared/charts/TransactionCharts/BrowserLineChart.tsx` uses it too.

Reply:
let me know what you think. https://github.com/elastic/kibana/pull/74931/commits/d2da9f181b4b915bff8a740f08717dec1018bde6